github/resource_github_branch_protection_test.go

```diff
@@ -556,6 +556,117 @@ func TestAccGithubBranchProtection(t *testing.T) {
 
 	})
 
+	t.Run("configures non-empty list of force push bypassers", func(t *testing.T) {
```

Comment:
Confirming that the newly added integration test succeeds for me locally. Some of the other tests are failing on main as well as this branch, which are unrelated to this PR.

Reply:
Argh, this is now failing for me.